.github/workflows/makefile.yml

```diff
@@ -0,0 +1,88 @@
+name: Makefile CI
+
+on:
+  push:
+    branches: [ "trunk" ]
+  pull_request:
+    branches: [ "trunk" ]
+
+env:
+  OPENSSL_VERSION: 3.5.0
+  APR_VERSION: 1.7.6
+  NASM_VERSION: 2.16.03
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Default
+            triplet: x64-windows
+            arch: x64
+            build-arch: X64
```

Comment:
Wouldn't it make sense to have only one of them?

Reply:
call "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\Tools\VsDevCmd.bat" -arch=X64
gives:
ERROR: Invalid host architecture 'X64'.
[ERROR:winsdk.bat] Host architecture is not valid : 'X64'

that is why 2 are used.